modules/process/pages/managing-dependencies.adoc

```diff
@@ -55,6 +55,33 @@ If you want to develop custom extensions for your processes, it is highly recomm
 
 This will help Maven selecting the matching versions if you are using a dependency already available in the Bonita runtime. When it is the case a warning should be displayed by your IDE. You can remove the version and set the scope to _provided_.
 
+== Troubleshooting 
+
+=== Missing dependencies in my configurations after a project migration
+
+When a project is migrated the target runtime version is updated and some provided dependencies may changes (version update or dependencies removed). 
+In some cases, one of your extension may have missing transitive dependency because the provided version provided by the runtime is not the same as the one expected by the extension. You can mitigate this issue by:
+
+* Removing this dependency from your configurations. The runtime dependency version will be used at runtime.
+* Explicitly add this dependency in your extensions with the required version (Project Overview > Extensions > Add a custom extension > Other...).
+
+[WARNING]
+====
+Both workarounds are not ideal.
+
+In the first case their is a risk that the version provided by the runtime is not compatible with the extension. You will also be more at risk with further version update if the provided dependency is updated/removed as it will be only be detect at runtime.
```

Comment:
```suggestion
In the first case there is a risk that the version provided by the runtime is not compatible with the extension. You will also be more at risk with further version update if the provided dependency is updated/removed as it will be only be detect at runtime.
```